narwhals/series.py

```diff
@@ -354,6 +354,34 @@ def mean(self) -> Any:
         """
         return self._compliant_series.mean()
 
+    def count(self) -> Any:
+        """
+        Returns the number of non-null elements in the Series.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> s = [1, 2, 3]
+            >>> s_pd = pd.Series(s)
+            >>> s_pl = pl.Series(s)
+
+            We define a library agnostic function:
+
+            >>> @nw.narwhalify
+            ... def func(s_any):
+            ...     return s_any.count()
+
+            We can then pass either pandas or Polars to `func`:
+
+            >>> func(s_pd)  # doctest:+SKIP
```

Comment:
why do we have to doctest skip this one? does it work without the skip?

Reply:
It's because of the numpy changes, as the new version causes pandas to output differently i.e np.int64(3), and causes the newer python tests to pass and older one's to fail.

Reply:
aah I see thanks - OK, tests are passing like this, let's ship it 🚀